pkg/metrics/metrics.go

```diff
@@ -44,6 +44,13 @@ var (
 		Buckets:   []float64{1, 5, 10, 15, 30, 60, 120, 300, 600, 1800, 3600},
 	}, []string{"operation"})
 
+	ExternalAPICalls = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: promNSUpjet,
+		Subsystem: promSysResource,
+		Name:      "external_api_calls",
```

Comment:
Because the external API time metric is named as `ext_api_duration`, should we name the new external API call counter metric as `ext_api_calls` instead of `external_api_calls`? Or something like `ext_api_calls_counter`?